scripting.md

```diff
@@ -0,0 +1,104 @@
+# SuperTux Level Editor Scripting Tutorial
+
+## What is Scripting?
+
+Scripting in SuperTux is when a game object is altered using things like a script trigger or a switch.  Scripting is written in the Squirrel coding language.
+
+#### What sort of things use scripting?
+
+Scripts can be activated by the following things, plus many more:
+
+* Script Trigger, which activates the script.
+* Switch/Button, which activate scripts when used.
+* Death Scripts, which activate when an enemy dies.
+* Init-Scripts, which activate when Tux enters the sector.
+
+In order to edit a script, right click a scriptable object and open the script editor.
+
+## Structure of scripts
+
+The basic structure of a line of script is as so:
+
+    object.action(Number);
+
+"object" is what you are going to activate, "action" is what the object does, and "Number" is the property of the action, like a number, series of numbers, or words. The semicolon activates the script, you have to use it in most situations.
+
+### Some scriptable objects
+
+Here are some things you can script in a sector:
+
+* Light: 
+
+      settings.set_ambient_light(color1, color2, color3);
+
+ where "color" is a decimal number between 0 and 1.
+
+* Music: 
+
+      play_music(\"music/song\"); 
+
+where "song" is a piece of music from the music folder.
+
+* A change to another sector's spawnpoint, or a spawnpoint in the current sector, is as shown: 
+
+      Level.spawn(\"sector\", \"spawn\");
+
+where "sector" is the sector and "spawn" is the spawnpoint.
+
+
+Are you starting to get it?
+
+### Cutscenes
+
+Here's an example of a cutscene, followed by the ending of a level.
+
+    Text.set_text(\"Tux: Penny? Are you here? \\nPenny: Over here!\\nI'm coming!\");
+
+	Text.set_centered(false);
+	
+	Text.fade_in(0.2);
+	
+	wait(4);
+	
+	Text.fade_out(0.2);
+	
+	wait(1.5);
+
+This would make a blob of text show around Tux that says:
+
+> "Tux: Penny? Are you here?
+> Penny: Over here!
+> I'm coming!"
+
+The slashes represent the text and the n represents a new line.
+
+wait(); represents how long to wait for the next script. Fading In and Out would take 0.2 seconds each and the text is not centered, according to the script.
+
+### End of a Level
```

Comment:
Nor is this a useful part, in my opinion.

Reply:
it is to me but i got rid of it.